lib/config.js

```diff
@@ -22,56 +29,54 @@ function parse(val) {
 
 // if the working directory contains a config.json, inherit from it.
 try {
-  var configJsonPath = path.resolve('config.json');
+  const configJsonPath = path.resolve('config.json');
   if (fs.existsSync(configJsonPath)) {
     console.log('Reading config from ' + configJsonPath);
-    var configJson = fs.readFileSync(configJsonPath, {encoding: 'utf8'});
-    var configObj = JSON.parse(configJson);
-    for (var key in configObj) {
+    const configJson = fs.readFileSync(configJsonPath, { encoding: 'utf8' });
+    const configObj = JSON.parse(configJson);
+    for (const key in configObj) {
```

Comment:
Yes, because for-in and for-of loops don't re-assign to the variable, but instead create a new, independent variable for each loop iteration. Classic for-loops re-assign to the variable though, so you have to use let in `for (let i = 0; i < 10; i++)`.

```jsshell
>>> o = {a: 1, b: 2, c: 3}
>>> for (const k in o) { console.log(k, o[k]); }
a 1
b 2
c 3
```